File-Formats/PDF/Converting-XPS-to-PDF.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Working with XPS document to PDF Conversion | Syncfusion
+description: This section explains how to converting XPS document to PDF document by using Syncfusion .NET PDF library.
+platform: file-formats
+control: PDF
+documentation: UG
+---
+# Converting XPS document to PDF
+
+The XPS (XML Paper Specification) document format is a fixed document format which consists of structured XML markup that defines the layout of a document and the visual appearance of each page, along with rendering rules for distributing, archiving, rendering, processing and printing the documents.
+
+The following code example explains how to converting XPS document to PDF using [XPSToPdfConverter](https://help.syncfusion.com/cr/file-formats/Syncfusion.XPS.XPSToPdfConverter.html) class.
```

Comment:
The following code example illustrates how to convert XPS to PDF ...

Reply:
Resolved